backend/routes/score.ts

```diff
@@ -0,0 +1,228 @@
+                                    saveToWorldLeaderboards(world, savedScore);
+
+                                    // a different way to save to leaderboards
+                                    // const leaderboards: Leaderboard[] = [];
+                                    //
+                                    // for (let i = 0; i < 4; i++) {
+                                    //     let leaderboard = leaderboardRepository
+                                    //         .findOneBy({ continent: world, daily: false })
+                                    //         .then((leaderboard) => {
+                                    //             if (leaderboard) {
+                                    //                 leaderboards.push(leaderboard);
+                                    //             }
+                                    //         });
+                                    // }
+
+
+                                    repository
+                                        .save(savedScore)
+                                        .then((score) => {
+                                            res.status(200).json(score);
+                                        })
+                                        .catch((error) =>
+                                            res
+                                                .status(404)
+                                                .json({ errorMessage: 'Score repo could not be found', error: error }),
+                                        );
+                                    // else return error in score
+                                }
+                            }
+                        });
+                    } else {
+                        res.statusMessage = 'Account not found';
+                        res.status(404).end();
+                        return;
+                    }
+                }
+            }
+        });
+    }
+});
+
+/*
+scoreRoute.get(baseUrl, (_req, res) => {
+
+    repository
+        .find({
+            relations: ['leaderboards', 'accounts'],
+            order: { scores: 'DESC' },
+            take:10,
+
+        })
+        .then((scores) => {
+            res.status(200).json(scores);
+        })
+        .catch((error) =>
+            res.status(404).json({ errorMessage: 'Could not be found', error: error }),
+        );
+});
+
+scoreRoute.get('/scores/player/:displayName', (req, res) => {
+    const userName = (req.params.displayName as string).toLowerCase();
+    console.log(userName);
+
+    repository
+        .find({
+            relations: ['accounts'],
+            where: { accounts: { displayName: userName } },
+        })
+        .then((scores) => {
+            res.status(200).json(scores);
+        })
+        .catch((error) =>
+            res.status(404).json({ errorMessage: `${userName} doesn't exist`, error: error }),
+        );
+});
```

Comment:
Remove commented area and push without. Then add it again to your local version so you still have it but we don't